helper/src/main/java/me/lucko/helper/plugin/ExtendedJavaPlugin.java

```diff
@@ -59,7 +59,9 @@
 
     // Used by subclasses to perform logic for plugin load/enable/disable.
     protected void load() {}
+
     protected void enable() {}
+
```

Comment:
IJ loves to automatically remove whitespace. Honestly, this is going to happen at some point in the future anyways, so we may as well get it out of the way.